razorpay/controllers/front/validation.php

```diff
@@ -16,60 +25,22 @@ public function postProcess()
 
         $amount = number_format($cart->getOrderTotal(true, 3), 2, '.', '')*100;
 
+        $api = new \Razorpay\Api\Api($key_id, $key_secret);
+
         $success = false;
-        $error = "";
-
-        try {
-            $url = 'https://api.razorpay.com/v1/payments/'.$razorpay_payment_id.'/capture';
-            $fields_string="amount=$amount";
-
-            //cURL Request
-            $ch = curl_init();
-
-            //set the url, number of POST vars, POST data
-            curl_setopt($ch,CURLOPT_URL, $url);
-            curl_setopt($ch,CURLOPT_USERPWD, $key_id . ":" . $key_secret);
-            curl_setopt($ch,CURLOPT_TIMEOUT, 60);
-            curl_setopt($ch,CURLOPT_POST, 1);
-            curl_setopt($ch,CURLOPT_POSTFIELDS, $fields_string);
-            curl_setopt($ch,CURLOPT_RETURNTRANSFER, TRUE);
-
-            //execute post
-            $result = curl_exec($ch);
-            $http_status = curl_getinfo($ch, CURLINFO_HTTP_CODE);
-
-
-            if($result === false) {
-                $success = false;
-                $error = 'Curl error: ' . curl_error($ch);
-            }
-            else {
-                $response_array = Tools::jsonDecode($result, true);
-                //Check success response
-                if($http_status === 200 and isset($response_array['error']) === false){
-                    $success = true;
-                }
-                else {
-                    $success = false;
-
-                    if(!empty($response_array['error']['code'])) {
-                        $error = $response_array['error']['code'].":".$response_array['error']['description'];
-                    }
-                    else {
-                        $error = "RAZORPAY_ERROR: Invalid Response <br/>".$result;
-                    }
-                }
-            }
-
-            //close connection
-            curl_close($ch);
+
+        try
```

Comment:
once https://github.com/razorpay/razorpay-php/pull/27 is merged, we can drop the try/catch and the success variable as well.